# HG changeset patch
# User Rik <rik@octave.org>
# Date 1480551869 28800
#      Wed Nov 30 16:24:29 2016 -0800
# Branch stable
# Node ID 6eb581b597bc395d4f135d7e3ce423f0b1eec71d
# Parent  5e111d533c99ad5a0c16eec5df34755ecfd7cad2
doc: Add missing change from cset 5e111d533c99

* sub2ind.cc (Fsub2ind): Indent @result macro in @example block.

diff --git a/libinterp/corefcn/sub2ind.cc b/libinterp/corefcn/sub2ind.cc
--- a/libinterp/corefcn/sub2ind.cc
+++ b/libinterp/corefcn/sub2ind.cc
@@ -95,17 +95,17 @@ The following example shows how to conve
 @code{(2,1)} and @code{(2,3)} of a 3-by-3 matrix to linear indices with a
 single call to @code{sub2ind}.
 
 @example
 @group
 s1 = [2, 2];
 s2 = [1, 3];
 ind = sub2ind ([3, 3], s1, s2)
-@result{} ind =  2   8
+    @result{} ind =  2   8
 @end group
 @end example
 @seealso{ind2sub, size}
 @end deftypefn */)
 {
   int nargin = args.length ();
 
   if (nargin < 2)
